src/main/java/com/gmail/goosius/siegewar/playeractions/PeacefullySubvertTown.java

```diff
@@ -97,9 +97,6 @@ private static void allowSubversionOrThrow(Player player, Nation residentsNation
 		if(SiegeController.hasActiveSiege(targetTown))
 			throw new TownyException(translator.of("msg_err_cannot_change_occupation_of_besieged_town"));
 
-		if(targetTown.hasNation() && targetTown.getNationOrNull() == residentsNation)
```

Comment:
This seems out of place here.

Reply:
- This is a small bug fix 
- Because a new requirement is that nations should be able to subvert their own towns.